### ISSUE TITLE
Backspace sends ^H (CTRL-H)

### ISSUE THREAD

**Issue by lifepillar** [terrychou/iVim#17]
As the subject says, deleting a character sends `^H`. Is that intended? Shouldn't it send `<BS>`?

**Comment by terrychou** [comment]
Please be more specific. Under what situation this happens? How can it be reproduced? How did you realize it as a problem? etc. More detailed information would help solve it faster.

**Comment by lifepillar** [comment]
To reproduce, type Ctrl-V followed by backspace in Insert mode. In terminal Vim, I get `^?`, but in iVim I get `^H`. In MacVim, I get `<BS>` (that's four characters), but I think that the character code is the same as in terminal Vim.

Character info can be obtained by placing the cursor over the character and typing `ga`. I get the following results, respectively:

```
<^?>  127,  Hex 7f,  Octal 177
<^H>  8,  Hex 08,  Octal 010
```

This prevents, for example, to use backspace in CtrlP (it moves the cursor to the left instead of deleting a character). Also, my µcomplete plugin binds `<c-h>` in Insert mode and that also breaks in iVim.

**Comment by terrychou** [comment]
Thank you for the information, I will look into it.

**Comment by terrychou** [comment]
Fixed in version 1.12
